### PR TITLE
Add files for PR labeler

### PR DESCRIPTION
**Pull request by Timthetic**
Fixes: #38

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+- any: ['src/components/PrizePage/**/*', 'src/pages/prizes.js']
+
+page:
+- any: ['src/components/MenuBar/**/*', 'src/components/Footer/**/*', 'src/components/layout.js']
+
+gallery:
+- any: ['src/components/GalleryPage/**/*', 'src/pages/gallery.js']
+
+infra:
+- any: ['.github/**/*', '!.github/FUNDING.yml']
```
Comment (TimothyGu) [inline .github/labeler.yml:17]: Let's also add these to `infra`
```
src/components/SvgImg.js
.editorconfig
.eslint*
.huskyrc.js
.nvmrc
.prettier*
.svgo*
gatsby-*.js
package.json
yarn.lock
```

And the following to `documentation`:
```
LICENSE*
*.md
.github/FUNDING.yml
```

Finally, make sure you add a new line to the end of files.

File: .github/labeler.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+- any: ['src/components/SchedulePage/**/*', 'src/pages/schedule.js']
+
+homepage:
+- any: ['src/components/HomePage/**/*', 'src/pages/index.js']
+
+prizes:
+- any: ['src/components/PrizePage/**/*', 'src/pages/prizes.js']
+
+page:
+- any: ['src/components/MenuBar/**/*', 'src/components/Footer/**/*', 'src/components/layout.js']
```
Comment (TimothyGu) [inline .github/labeler.yml:11]: Also `src/components/seo.js`

File: .github/labeler.yml (hunk 1 above)
Comment (TimothyGu) [inline .github/labeler.yml:17]: Another thing is that I think you meant to say "anything in `.github`, except `FUNDING.yml`". If this is the case, then you should do

```yaml
infra:
- any: ['.github/**/*']
  all: ['!.github/FUNDING.yml']
```

See https://github.com/actions/labeler#common-examples

File: .github/labeler.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+schedule:
+- any: ['src/components/SchedulePage/**/*', 'src/pages/schedule.js']
```
Comment (TimothyGu) [inline .github/labeler.yml:2]: This probably doesn't do what you want it to do. https://github.com/actions/labeler#match-object says that `any` means

> match ALL globs against ANY changed path.

It's obviously impossible for any changed path to match both of these globs.

Instead, you should do this:

```yaml
schedule:
- src/components/SchedulePage/**/*
- src/pages/schedule.js
```

Same goes for everything else.

File: .github/labeler.yml (hunk 1 above)
Comment (Timthetic) [reply .github/labeler.yml:17]: Wait, wouldn't that one be: "any change to `.github/**/*` files as long as the `.github/FUNDING.yml` hasn't changed"

Comment (TimothyGu) [reply .github/labeler.yml:17]: Ah, I think you're right. The status quo looks good then.

**Pull request merged**